building/build-debs/homeworld-admin-tools/src/version.py

```diff
@@ -14,5 +14,8 @@ def display_version():
     git_version = get_git_version()
     print("Git commit hash:", git_version)
 
+    apt_branch = get_resource("APT_BRANCH").decode().rstrip()
+    print("Apt branch:", apt_branch)
+
```

Comment:
Perhaps this should probably be refactored into a separate function, since it's used more than once?

Reply:
Good idea.